hack/generator/pkg/codegen/testdata/ArmResource/Arm_test_id_resource_reference_azure.golden

```diff
@@ -324,6 +324,10 @@ func (fakeResourceProperties *FakeResourceProperties) PopulateFromARM(owner genr
 		return fmt.Errorf("unexpected type supplied for PopulateFromARM() function. Expected FakeResourcePropertiesARM, got %T", armInput)
 	}
 
+	// no assignment for property ‘Reference’:
+
+	// no assignment for property ‘SubnetReference’:
```

Comment:
Really minor, but maybe these don't need the trailing `:` #trivial #nitpick

Reply:
Fixed